src/dualscreen.tsx

```diff
@@ -342,17 +367,23 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
           const ks: string = this._player.config.session.ks;
           if (!entryId) {
             this.logger.warn('Secondary entry id not found');
+            // subscribe on timed metadata events for image player
```

Comment:
this logic you should do also when config.secondaryPlayerType is image, regardless of the entry id existence

Reply:
we don't need anymore config.secondaryPlayerType so since it's redundant. Dual-screen plugin get's information about secondaryPlayerType from API response